### ISSUE TITLE
Fails to Start Minecraft with 0.4.0-alpha6+build.14

### ISSUE THREAD

**Issue by Lark2Bird** [jellysquid3/Sodium#1141]
### Version information

0.4.0-alpha6+build.14

### Reproduction Steps

Attempt to launch Minecraft using only Fabric API (v0.46.1+1.18) with Sodium (v0.4.0-alpha6+build.14). Game fails to launch.

### Crash Report file

[2022-01-17-6.log](https://github.com/CaffeineMC/sodium-fabric/files/7882783/2022-01-17-6.log)

**Comment by amnotbananaama** [comment]
The log you provided does not contain a crash, Minecraft loads normally. Is the game freezing, or crashing? If Minecraft is freezing while loading, this is probably a duplicate of #899. Try updating your graphics drivers as suggested by duplicates of that issue

**Comment by Lark2Bird** [comment]
So I just checked, and I have the latest version of my graphics drivers and everything else is up to date. 

I tried the native launcher just to make sure it wasn't a multi-mc issue, and it game me this in a fabric window "RuntimeException: Failed to read accessWidener file from mod sodium" in a stacktrace (attached). 

I was able to successfully launch MC with Fabric by itself, and also with Fabric API with no problems. The game does not give me a crash log as it never gets to that stage in the loading process from what it seems.

[Stacktrace.11822.txt](https://github.com/CaffeineMC/sodium-fabric/files/7893010/Stacktrace.11822.txt)

**Comment by magneticflux-** [comment]
You put the "dev" version in your mods folder. The "dev" version uses the `named` form of Minecraft (which is Minecraft with the Yarn (or other) names replacing intermediary names of classes, methods, variables, etc.). The regular version uses the `intermediary` form of Minecraft (which is Minecraft with the intermediary names replacing Mojang-obfuscated names of classes, methods, variables, etc.).

More info can be found here: https://fabricmc.net/wiki/tutorial:mappings

TLDR: you used the jar for development environments in a production environment. Use the jar without "-dev" at the end.

**Comment by Lark2Bird** [comment]
Totally makes sense. Can't believe I didn't catch it earlier. Thanks for your help!
